### PR TITLE
Allanbranch

### PR DESCRIPTION
**Pull request by allandx**
Included Devops logo at content page and included hyperlink of creating an MS Azure acc at README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Good luck.
 
 # EDIT_ME
 
+![devopslogo](/devopslogo.jpg)
```
Comment (ytbryan) [inline README.md]: For some reason, it's still not previewing....

Comment (ytbryan) [reply README.md]: CleanShot 2021-04-30 at 09 54 04

Comment (hadibolehjadi) [reply README.md]: hmm that's odd... yesterday we managed to get it to work while on Allan's machine. @allandx could you try it again on your machine to see if it is still displaying?

Comment (ytbryan) [reply README.md]: maybe remove the `/` 

can you also call it logo.jpg

**Comment by allandx** [comment]
Hi @ytbryan @hadibolehjadi , have made the changes in 02db65f as per @ytbryan comments. It is still able to show on my machine. Please see if it is able to be previewed on yours?

File: README.md (hunk 1 above)
Comment (ytbryan) [reply README.md]: ok cool it works now! @allandx

**Review by ytbryan — APPROVED** [review]

**Review by ytbryan — APPROVED** [review]

**Review by hadibolehjadi — APPROVED** [review]

**Pull request merged**